### ISSUE TITLE
training Loss is weird

### ISSUE THREAD

**Issue by renq-mt** [thuanz123/realfill#24]
It is my first time to fine-tune sd model. hence, i do have several problems about the result.
Why is the loss during the training process always 0? 
Is it because the reconstruction loss function converges quickly? 
In this condition， what has the model learn?
the loss function is：
loss = (weightings * F.mse_loss(model_pred.float(), noise.float(), reduction="none")).mean()

<img width="400" alt="image" src="https://github.com/user-attachments/assets/fb26a8d0-db22-4e70-bac1-3ad6bf291fe5">

**Comment by thuanz123** [comment]
Are you using your own data or the example data ?

**Comment by thuanz123** [comment]
if you are using example data then you have changed something in the code that breaks the algorithm. If you use your own data then you probably have wrong masking or incorrect data. For the later case, you can try to match the example data [here](https://github.com/thuanz123/realfill/tree/main/data/flowerwoman).
